x/ownership/internal/keeper/keeper.go

```diff
@@ -77,12 +82,29 @@ func (k Keeper) Delete(ctx sdk.Context, owner sdk.AccAddress, resourceHash hash.
 		return fmt.Errorf("resource %q do not have any ownership", resourceHash)
 	}
 
+	// transfer all spendable coins from resource address to owner
+	addr := sdk.AccAddress(crypto.AddressHash(resourceHash))
+	coins := k.bankKeeper.GetCoins(ctx, addr)
+	if err := k.bankKeeper.SendCoins(ctx, addr, owner, coins); err != nil {
```

Comment:
Are we sure that this will not raise an error in case the resource (eg: runner) didn't receive any funds.
If so then we have an issue and cannot delete a runner that we just started or was never used.

Reply:
> Are we sure that this will not raise an error in case the resource (eg: runner) didn't receive any funds.

What type of error? We must be able to send coins. If you are asking if there is no coins on runner acc, then send coins will return immediately.

Reply:
Could you make sure that the following flow works:
- Start a runner for a specific service
- Don't create any executions for this runner
- Stop the runner
This should be possible with no error, I'm just not sure here this is possible because `SendCoins` will return an error and so we will not be able to stop a runner.

Reply:
Fix the bug, nice catch 😎